### PR TITLE
Implementation Support Plan for Micro-lesson

### PR DESCRIPTION
**Pull request by ttimbers**
First draft of an implementation plan to get the Micro-lessons happening. Please feel free to edit/suggest/comment.

### CODE REVIEW

File: subcommittees/mentoring/Micro-Lesson_Implementation.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+7. The Mentor will then recommend to the Software Carpentry Steering Committee whether
+each trainees should now become a badged instructor or whether they need teach another 
+Micro-lesson before they become a badged instructor.
+
+
+### Questions to ask at the group debrief:
+
+* How many learners? 
+* What topic was taught? Shell, Git, Python, R, MATLAB or SQL?
```
Comment (r-gaia-cs) [inline subcommittees/mentoring/Micro-Lesson_Implementation.md]: Drop "Shell, Git, Python, R, MATLAB or SQL?".

File: subcommittees/mentoring/Micro-Lesson_Implementation.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
+## Implementation Support Plan for Micro-lesson
+
+Below is an implementation support plan for the proposed Micro-Lesson to be taught by 
+those who have recently completed instructor training but not yet taught a Software 
+Carpentry workshop. After successfully completing the workflow outlined below, the student
+can then graduate and become a badged Software Carpentry instructor.
+
+
+Estimating an average of 40 people completing instructor training, and that all of them
```
Comment (r-gaia-cs) [inline subcommittees/mentoring/Micro-Lesson_Implementation.md]: From data that @gvwilson provide me we had 285 only in the first half of this year. We need to rethink the numbers.

As we discussed on June 12th, maybe we could run this four times/year (once for each season).

**Comment by ttimbers** [comment]
Updated micro-lesson implementation plan. I tried to incorporate the mentoring committees feedback.

**Comment by ttimbers** [comment]
@rgaiacs do we want to include the mini-lesson implementation doc we wrote? I sent this PR to to board in July but it was never merged.

**Comment by rgaiacs** [comment]
> do we want to include the mini-lesson implementation doc we wrote?

Sure. I just need to give another round of review.

> I sent this PR to to board in July but it was never merged.

Sorry for the delay.

File: subcommittees/mentoring/Micro-Lesson_Implementation.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
+## Implementation Support Plan for Micro-lesson
+
+Below is an implementation support plan for the proposed Micro-Lesson to be taught by 
+those who have recently completed instructor training but not yet taught a Software 
+Carpentry workshop. The micro-lesson assignment outlined it to be done after instructor
+training has been completed. After successfully completing the workflow outlined below, 
+the student can then graduate and become a badged Software Carpentry instructor.
```
Comment (rgaiacs) [inline subcommittees/mentoring/Micro-Lesson_Implementation.md]: I'm just waiting that @gvwilson confirm this to me.

File: subcommittees/mentoring/Micro-Lesson_Implementation.md [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+
+### Mentor Requirements
+Estimating an average of 40 people completing instructor training, and that all of them
+choose to participate in the Micro-lesson to obtain their Software Carpentry Instructor
```
Comment (rgaiacs) [inline subcommittees/mentoring/Micro-Lesson_Implementation.md]: Again waiting confirmation on this.

File: subcommittees/mentoring/Micro-Lesson_Implementation.md [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+Estimating an average of 40 people completing instructor training, and that all of them
+choose to participate in the Micro-lesson to obtain their Software Carpentry Instructor 
+badge, the program outlined below will require, at most, 8 Mentors (Software Carpentry 
+Instructors who have taught 3 or more lessons in the previous year). Each Mentor will be 
+responsible for a maximum of 5 individuals who have recently completed instructor 
+training, herein referred to as trainees. For this program to be successful, each Mentor 
+will have to commit a maximum of ~ 9 - 10 hours over 3 months. 
+
+### Program frequency and capacity
+The Micro-lesson program will be run 3 times a year (each program spanning 3 months), and
```
Comment (rgaiacs) [inline subcommittees/mentoring/Micro-Lesson_Implementation.md]: Can we reword this to accommodate the pilot at the begin of 2016?

**Comment by rgaiacs** [comment]
@ttimbers Thanks for this pull request. I'm closing it because it was supersede by #65.
